web_src/css/explore.css

```diff
@@ -10,6 +10,28 @@
   margin-right: 5px;
 }
 
+.explore .filter .ui.right .menu {
+  max-height: 60vh;
+  overflow-y: auto;
+}
+
+.explore .filter .ui.dropdown {
+  float: right;
+}
```

Comment:
Some unique classnames would be beneficial here as well. We should not tie CSS so tightly to HTML structure, because HTML structure can always change during refactoring, which is where these selectors break.

Reply:
done :)